src/components/share-overlay/share-overlay.js

```diff
@@ -476,6 +476,26 @@ class ShareOverlay extends Component {
     this.props.setIsModal(true);
   }
 
+  /**
+   * update url with params and values
+   *
+   * @param {string} url - the share url
+   * @param {string} key - the param name
+   * @param {string} value - the param value
+   * @returns {string} - share url with the query parameters
+   * @memberof ShareOverlay
+   */
+  _updateUrlParams(url: string, key: string, value: number): string {
+    const urlObj = new URL(url);
```

Comment:
i like the use of the URL object, which implements some logic for you.
note, though, that if new URL() gets an argument which is not a valid url, it will throw an exception.
is the URL constructor argument url in this case guaranteed to be a valid url ?

(URL.canParse actually checks for validity, but it is not supported by all browsers)

Reply:
done